arcgis-ios-sdk-samples/Search/Offline geocode/OfflineGeocodeViewController.swift

```diff
@@ -117,7 +117,7 @@ class GeocodeOfflineViewController: UIViewController, AGSGeoViewTouchDelegate, U
         })
     }
     
-    private func reverseGeocode(point:AGSPoint) {
+    fileprivate func reverseGeocode(_ point:AGSPoint) {
```

Comment:
similar to other comments above about having argument name to clarify call site usage